src/Target.h

```diff
@@ -139,7 +141,7 @@ struct Target {
      * OpenGL, because it is not capable of gpgpu, and is not
      * scheduled via Func::gpu_tile. */
     bool has_gpu_feature() const {
-        return has_feature(CUDA) || has_feature(OpenCL);
+        return has_feature(CUDA) || has_feature(OpenCL) || has_feature(Renderscript);
```

Comment:
I'm not sure why we won't allow gpu_ scheduling for Renderscript kernels? Currently Func::shader() uses gpu_blocks when it sets up scheduling for Renderscript.